services/spar/test-integration/Spec.hs

```diff
@@ -29,7 +29,7 @@ main :: IO ()
 main = do
   (wireArgs, hspecArgs) <- partitionArgs <$> getArgs
   env <- withArgs wireArgs mkEnvFromOptions
-  withArgs hspecArgs . hspec . beforeAll (pure env) . afterAll destroyEnv $ mkspec
+  withArgs (hspecArgs <> ["-m", "###"]) . hspec . beforeAll (pure env) . afterAll destroyEnv $ mkspec
```

Comment:
Looks like you can use [`focus`](https://hackage.haskell.org/package/hspec-2.7.1/docs/Test-Hspec.html#v:focus) instead.

Reply:
(appearently not in 2.5.5, though.  and i like my solution, too.  :))